### ISSUE TITLE
Native IPv6 implementation fails on Alpine Linux

### ISSUE THREAD

**Issue by ghost** [andryyy/mailcow-dockerized#4242]
Hi,
i am running my mailcow on a Alpine Linux host. With my ./update.sh run today, mailcow offered the native IPv6 implementation to me. The implementation crashed, since update.sh is using systemctl without checking, whether its available. On Alpine, there is no systemd - it is using openrc.

```
mx:/opt/mailcow-dockerized# ./update.sh 
Checking internet connection... OK
Checking for newer update script...
Updated 0 paths from a5b6ecac
Are you sure you want to update mailcow: dockerized? All containers will be stopped. [y/N] y
Native IPv6 implementation available.
This will enable experimental features in the Docker daemon and configure Docker to do the IPv6 NATing instead of ipv6nat-mailcow.
!!! This step is recommended !!!
mailcow will try to roll back the changes if starting Docker fails after modifying the daemon.json configuration file.
Should we try to enable the native IPv6 implementation in Docker now (recommended)? [y/N] y
Working on IPv6 NAT, please wait...
./update.sh: line 172: systemctl: command not found
Error: Failed to activate IPv6 NAT! Reverting and exiting.
./update.sh: line 175: systemctl: command not found
./update.sh: line 176: systemctl: command not found
Validating docker-compose stack configuration...
Checking for conflicting bridges...
Saving diff to update_diffs/diff_before_update_2021-08-26-21-26-34...
Prefetching images...
1.13: Pulling from mailcow/unbound
540db60ca938: Already exists 
5a36a397daac: Pull complete 
...
```

I will begin working on a solution - from my point of view, it shouldnt be that hard. I will create a PR in the next hours or maybe also days. 

This should be only a warning for other Alpine Linux users.

**Comment by ghost** [comment]
I have built a possible fix. But i can't test it on my mailcow instance, since the update.sh is everytime pulling the latest version from github master and overrides itself. Even with --ours it doesn't work.

May somebody can give me a hint, how i can test this?

**Comment by MAGICCC** [comment]
You could hack something in these lines:
https://github.com/mailcow/mailcow-dockerized/blob/master/update.sh#L525-L534

**Comment by ghost** [comment]
> You could hack something in these lines:
> https://github.com/mailcow/mailcow-dockerized/blob/master/update.sh#L525-L534

This was the solution. Thank you!
PR will follow.

**Comment by ghost** [comment]
Fixed due to merge of #4244
